package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ts-proto",
-  "version": "1.20.2",
+  "version": "1.21.0",
```

Comment:
Technically this is a new feature vs a bugfix, so I went with 1.21.0 vs 1.20.3 -- I'm ambivalent though.